Assets/_Scenes/MainScene.unity

```diff
@@ -203,6 +203,80 @@ Transform:
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 2
+--- !u!1 &614197765
```

Comment:
Ah if 'UnityEditor' is not available, which path should I use to save the screenshot?